crypto/bn/bn_exp.c

```diff
@@ -393,7 +392,6 @@ int BN_mod_exp_mont(BIGNUM *rr, const BIGNUM *a, const BIGNUM *p,
                                  * when there is only the value '1' in the
                                  * buffer. */
     wstart = bits - 1;          /* The top bit of the window */
-    wend = 0;                   /* The bottom bit of the window */
 
 #if 1                           /* by Shay Gueron's suggestion */
```

Comment:
If we're removing redundant code, the `#if 1` and its accompanying `#endif` would be good to remove too

Reply:
Moved the comment and removed `#if 1 #endif` in an additional commit